### ISSUE TITLE
Move card window randomly closes during move process

### ISSUE THREAD

**Issue by yougotborked** [nextcloud/deck#5404]
**Describe the bug**
The Move Card dialog box randomly seems to close itself

**To Reproduce**
1. Go to any board and any card
2. Attempt to move the card to another board

Another user on my server is also experiencing this on a windows 11 edge browser.

**Expected behavior**
The dialog does not close until I click cancel

**Screenshots**
This is a recording of me trying to move a card. I'm not clicking anything when the dialog closes itself.
![card-goes-away](https://github.com/nextcloud/deck/assets/1314122/01c25635-331a-4f06-bd8f-a337874e793e)

**Client details:**
 - OS: Windows 11
 - Firefox 121
 - Nextcloud 28.0.1
 - Device: desktop

<details>
<summary>Server details</summary>

**Operating system**: Fedora 39 running the official docker image via compose, reverse proxy with apache, postgres db.

**Web server:**

**Database:**

**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)

**Where did you install Nextcloud from:**

**Signing status:**

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.

no errors here when I did thiss
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```

**Are you using an external user-backend: no

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
nothing notable during the interaction
```

#### Browser log
I don't know if this is helpful but this is what I saw while I did the action
![image](https://github.com/nextcloud/deck/assets/1314122/e4495687-4169-481b-960e-b0edfee63a33)

</details>

**Comment by Aaron-von-Awesome** [comment]
I am experiencing the same thing.  I am unable to move cards via the web UI; I have to use the Deck Android app in order to move cards to a different board.

**Comment by mattheskay** [comment]
Me too, is there any workaround if you don´t use android? The iOS apps available don´t support this function at all!

**Comment by Jason-Linux** [comment]
Hi Same Things

{
    "reqId": "ZZfHqvAvP5yAm9rjstG9yAAAAA0",
    "level": 3,
    "time": "2024-01-05T09:11:07+00:00",
    "remoteAddr": "",
    "user": "",
    "app": "no app in context",
    "method": "PROPFIND",
    "url": "/remote.php/webdav",
    "message": "Error while getting file info",
    "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:121.0) Gecko/20100101 Firefox/121.0",
    "version": "28.0.1.1",
    "exception": {
        "Exception": "Icewind\\SMB\\Exception\\ForbiddenException",
        "Message": "Invalid request for / (ForbiddenException)",
        "Code": 1,
        "Trace": [
            {
                "file": "/nextcloud/apps/files_external/3rdparty/icewind/smb/src/Native/NativeState.php",
                "line": 75,
                "function": "fromMap",
                "class": "Icewind\\SMB\\Exception\\Exception",
                "type": "::"
            },
            {
                "file": "/nextcloud/apps/files_external/3rdparty/icewind/smb/src/Native/NativeState.php",
                "line": 92,
                "function": "handleError",
                "class": "Icewind\\SMB\\Native\\NativeState",
                "type": "->"
            },
            {
                "file": "/nextcloud/apps/files_external/3rdparty/icewind/smb/src/Native/NativeState.php",
                "line": 351,
                "function": "testResult",
                "class": "Icewind\\SMB\\Native\\NativeState",
                "type": "->"
            },
            {
                "file": "/nextcloud/apps/files_external/3rdparty/icewind/smb/src/Native/NativeShare.php",
                "line": 305,
                "function": "getxattr",
                "class": "Icewind\\SMB\\Native\\NativeState",
                "type": "->"
            },
            {
                "file": "/nextcloud/apps/files_external/3rdparty/icewind/smb/src/Native/NativeFileInfo.php",
                "line": 43,
                "function": "getAttribute",
                "class": "Icewind\\SMB\\Native\\NativeShare",
                "type": "->"
            },
            {
                "file": "/nextcloud/apps/files_external/3rdparty/icewind/smb/src/Native/NativeFileInfo.php",
                "line": 69,
                "function": "stat",
                "class": "Icewind\\SMB\\Native\\NativeFileInfo",
                "type": "->"
            },
            {
                "file": "/nextcloud/apps/files_external/3rdparty/icewind/smb/src/Native/NativeShare.php",
                "line": 114,
                "function": "getSize",
                "class": "Icewind\\SMB\\Native\\NativeFileInfo",
                "type": "->"
            },
            {
                "file": "/nextcloud/apps/files_external/lib/Lib/Storage/SMB.php",
                "line": 201,
                "function": "stat",
                "class": "Icewind\\SMB\\Native\\NativeShare",
                "type": "->"
            },
            {
                "file": "/nextcloud/apps/files_external/lib/Lib/Storage/SMB.php",
                "line": 587,
                "function": "getFileInfo",
                "class": "OCA\\Files_External\\Lib\\Storage\\SMB",
                "type": "->"
            },
            {
                "file": "/nextcloud/lib/private/Files/Storage/Wrapper/Wrapper.php",
                "line": 596,
                "function": "getMetaData",
                "class": "OCA\\Files_External\\Lib\\Storage\\SMB",
                "type": "->"
            },
            {
                "file": "/nextcloud/lib/private/Files/Storage/Wrapper/PermissionsMask.php",
                "line": 140,
                "function": "getMetaData",
                "class": "OC\\Files\\Storage\\Wrapper\\Wrapper",
                "type": "->"
            },
            {
                "file": "/nextcloud/lib/private/Files/Storage/Wrapper/Wrapper.php",
                "line": 596,
                "function": "getMetaData",
                "class": "OC\\Files\\Storage\\Wrapper\\PermissionsMask",
                "type": "->"
            },
            {
                "file": "/nextcloud/lib/private/Files/Cache/Scanner.php",
                "line": 121,
                "function": "getMetaData",
                "class": "OC\\Files\\Storage\\Wrapper\\Wrapper",
                "type": "->"
            },
            {
                "file": "/nextcloud/lib/private/Files/Cache/Scanner.php",
                "line": 158,
                "function": "getData",
                "class": "OC\\Files\\Cache\\Scanner",
                "type": "->"
            },
            {
                "file": "/nextcloud/lib/private/Files/View.php",
                "line": 1489,
                "function": "scanFile",
                "class": "OC\\Files\\Cache\\Scanner",
                "type": "->"
            },
            {
                "file": "/nextcloud/lib/private/Files/Node/Folder.php",
                "line": 102,
                "function": "getDirectoryContent",
                "class": "OC\\Files\\View",
                "type": "->"
            },
            {
                "file": "/nextcloud/apps/dav/lib/Connector/Sabre/Directory.php",
                "line": 261,
                "function": "getDirectoryListing",
                "class": "OC\\Files\\Node\\Folder",
                "type": "->"
            },
            {
                "file": "/nextcloud/3rdparty/sabre/dav/lib/DAV/Tree.php",
                "line": 206,
                "function": "getChildren",
                "class": "OCA\\DAV\\Connector\\Sabre\\Directory",
                "type": "->"
            },
            {
                "file": "/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php",
                "line": 900,
                "function": "getChildren",
                "class": "Sabre\\DAV\\Tree",
                "type": "->"
            },
            {
                "file": "/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php",
                "line": 982,
                "function": "generatePathNodes",
                "class": "Sabre\\DAV\\Server",
                "type": "->"
            },
            {
                "file": "/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php",
                "line": 1662,
                "function": "getPropertiesIteratorForPath",
                "class": "Sabre\\DAV\\Server",
                "type": "->"
            },
            {
                "file": "/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php",
                "line": 1647,
                "function": "writeMultiStatus",
                "class": "Sabre\\DAV\\Server",
                "type": "->"
            },
            {
                "file": "/nextcloud/3rdparty/sabre/dav/lib/DAV/CorePlugin.php",
                "line": 346,
                "function": "generateMultiStatus",
                "class": "Sabre\\DAV\\Server",
                "type": "->"
            },
            {
                "file": "/nextcloud/3rdparty/sabre/event/lib/WildcardEmitterTrait.php",
                "line": 89,
                "function": "httpPropFind",
                "class": "Sabre\\DAV\\CorePlugin",
                "type": "->"
            },
            {
                "file": "/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php",
                "line": 472,
                "function": "emit",
                "class": "Sabre\\DAV\\Server",
                "type": "->"
            },
            {
                "file": "/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php",
                "line": 253,
                "function": "invokeMethod",
                "class": "Sabre\\DAV\\Server",
                "type": "->"
            },
            {
                "file": "/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php",
                "line": 321,
                "function": "start",
                "class": "Sabre\\DAV\\Server",
                "type": "->"
            },
            {
                "file": "/nextcloud/apps/dav/appinfo/v1/webdav.php",
                "line": 88,
                "function": "exec",
                "class": "Sabre\\DAV\\Server",
                "type": "->"
            },
            {
                "file": "/nextcloud/remote.php",
                "line": 172,
                "args": [
                    "/nextcloud/apps/dav/appinfo/v1/webdav.php"
                ],
                "function": "require_once"
            }
        ],
        "File": "/nextcloud/apps/files_external/3rdparty/icewind/smb/src/Exception/Exception.php",
        "Line": 44,
        "message": "Error while getting file info",
        "exception": [],
        "CustomMessage": "Error while getting file info"
    },
    "id": "6597c7b43682f"
}

I remove personnal info

**Comment by Jason-Linux** [comment]
After short time do this. same if we use firefox or chrome or edge

**Comment by sambila** [comment]
Hi, I have the same issue. With Arc as brosers on Apple M1, Ventura.

**Comment by viktorix** [comment]
I just wanted to add. The modal will close automatically after about 5 seconds of being opened. You don't have to do anything. It will just close.

**Comment by bentuna** [comment]
Can confirm that. The issue appears in both the modal inside the card modal as well as outside.

**Comment by juliushaertl** [comment]
@luka-nextcloud Could you have a look into that. I think this is due to the fact that the Modal was moved into the actions component https://github.com/nextcloud/deck/blob/b06b41f7ebed0b44e8d2f2b374813541f213d163/src/components/cards/CardMenuEntries.vue#L72 which causes it to get removed once the actions hide on click.

**Comment by juliushaertl** [comment]
Maybe we can use https://nextcloud-libraries.github.io/nextcloud-dialogs/functions/index.spawnDialog.html for this instead or move the modal to a top level component and trigger it with `@nextcloud/event-bus` events?

**Comment by ilsawa** [comment]
Wow!!! I did it. I was able to move the card to another board. This can be done if you make a choice very very quickly before the move window closes itself.
You just need to treat it as a reaction speed training :P

**Comment by donty** [comment]
I thought it was just when moving a card but it seems to be affecting other aspects of Deck too, eg attempting to create a new card can randomly open an existing one instead depending where the mouse was from what I can tell. Sort of a big problem as we can't use it properly at all.

**Comment by chris-meyer3** [comment]
Same problem here (NC 28.0.1)

**Comment by PeGa** [comment]
Same problem here, NC 28.0.2. Exactly as described. In my case, might not have headshot reflexes (should I install counter-strike again to get shape?), so I'm unable to move the card.

**Comment by chris-meyer3** [comment]
Any progress on this issue? Please 'review'.
TNX

**Comment by chris-meyer3** [comment]
Fixed in 28.0.3 - TNX!
